infra/fastly/terraform/dl.k8s.io/services.tf

```diff
@@ -0,0 +1,89 @@
+resource "fastly_service_vcl" "files" {
+  name     = "cdn.dl.k8s.io"
+  activate = true
+
+  domain {
+    name = "cdn.dl.k8s.io"
```

Comment:
I suggest `fastly.dl.k8s.io` - it sets up a dynamic where we ack the contributions of the CDNs, and thereby incentivize more CDNs to contribute bandwidth.

dl.k8s.io can then use archeio one day, and likely a simple DNS redirect in the short term

Reply:
+1, the rest looks pretty fine but let's call out fastly here. thanks @chrispoole643 🙏

Reply:
I think it's fine to use this domain for the moment. there is no direct benefict to use a different subdomain.
we can revisit that later.